src/dev.config.ts

```diff
@@ -30,6 +33,16 @@ function webpackConfig(args: any): webpack.Configuration {
 		);
 	}
 
+	if (includesExternals) {
+		config.plugins.push(
+			new ExternalLoaderPlugin({
+				dependencies: externalDependencies,
+				outputPath: args.externals && args.externals.outputPath,
+				pathPrefix: args.withTests ? '../_build/src' : ''
```

Comment:
is `withTests` a valid argument to cli-build-app? We just use the separate test conf when passed a mode of test now I think?

Reply:
I don’t think the build commands use `_build` for anything so I’m not totally sure what this does either?